.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,32 @@
-> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.
+# Add here a title for your PR
 
-### What does this PR do?
+<!-- 
+Please use one of the following prefixes for the title:
+docs: Documentation not including procedures. Engineering review is mandatory.
+procedures: Documentation including procedures. Testing procedures is mandatory. Engineering and QE review is mandatory (Engineering can review on behalf of QE). 
+chore: Choreography, release, tooling, version upgrades.
+fix: Fix build, language, links, or metadata.
+-->
 
+<!-- Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR. -->
 
-### What issues does this PR fix or reference?
+## What does this PR do?
 
+## What issues does this PR fix or reference?
 
-### Specify the version of the product this PR applies to.
+## Specify the version of the product this PR applies to
 
+## Pull Request Checklist
 
-### PR Checklist
+The author and the reviewers validate the content of this Pull Request with this checklist, in addition to the [automated tests](code_review_checklist.adoc).
 
-As the author of this Pull Request I made sure that:
-
-- [ ] `vale` has been run successfully against the PR branch
-- [ ] Link checker has been run successfully against the PR branch
-- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
-- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
+- Any procedure:
+  - [ ] Successfully tested.
+  - [ ] Describes a scenario already covered by QE tests, otherwise Che QE team has acknowledged an issue.
```

Comment:
I am afraid this will lead to creation of many issue that never be resolved.
I've already been in such situation: I had to make small change in docs, but such requirement forced me to create an issue for QE team which had no value (and the issue was closed by them afterwards). Another point, even if such issues will be resolved, it will increase: 1) Time spent by QE on supporting all the tests 2) QA instances workload. In my opinion there should be some balance: only new major features should have new integration tests.

Reply:
I agree. This sentence is *already there*, but hardly followed.  We have to find the right formulation to have something meaningful here.

Reply:
```suggestion
```